plotting/drawhits.py

```diff
@@ -180,6 +180,19 @@ def parse_args():
     layer_cells[ln] = cells
     n_tot_cells += cells
 
+print("Retrieve the number of sensors per module/cell (assuming this is the same everywhere within the layer")
+sensors_per_module_map = {}
+try:
+    for det_element, sensors_per_module in detector_dict["sensors_per_module_map"].items():
+        ln = layer_number_from_string(det_element)
+        sensors_per_module_map[ln] = sensors_per_module
```

Comment:
This part could use the helper function `simplify_dict`, which should be achieving what you want here.
```suggestion
        sensors_per_module_map = simplify_dict(detector_dict["sensors_per_module_map"])
```

Reply:
Applied